migrations/2023-07-04-131721_add_org_id_and_org_name/up.sql

```diff
@@ -0,0 +1,3 @@
+-- Your SQL goes here
+ALTER TABLE merchant_account
+ADD COLUMN IF NOT EXISTS organization_id VARCHAR(32);
```

Comment:
https://github.com/juspay/hyperswitch/blob/c119bfdd7e93d345c340cf1282f47ab297b2c4e2/crates/common_utils/src/consts.rs#L4

This function will generate the id of length 20, this will be used when creating the organization id.